specs/protocol/isthmus/configurability.md

```diff
@@ -157,3 +216,28 @@ The following fields are included:
 - `version` is `uint256(0)`
 - `opaqueData` is the tightly packed transaction data where `mint` is `0`, `value` is `0`, the `gasLimit`
    is `200_000`, `isCreation` is `false` and the `data` is the data passed into `upgrade`.
+
+## Consensus Parameters
+
+The operator fee scalar and constant are new consensus parameters, so we define standard values for them.
+
+### [Operator Fee Scalar](exec-engine.md#operator-fees)
+
+**Description:** Operator fee scalar -- used to calculate the operator fee<br/>
+**Administrator:** [Operator Fee Manager](#operator-fee-manager)<br/>
+**Requirement:** Between 0 and 0.5 * (baseFee + priorityFee) <br/>
```

Comment:
This isn't noted anywhere, apologies about that, but its probably best to put these configurability requirements in [this](https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/configurability.md) file so they are all in a single place

Also we would like to say that the requirement for these values is 0 for now for the superchain, with op-succinct you can use them however you would like, then when we do add zk as part of the superchain's proof system we can define the ranges of values. We don't have bandwidth to really think deeply about what the standard values should be from a product perspective and while what you have now could make sense, I don't want to ratify something thru gov that ultimately isnt right